README.md

```diff
@@ -1,6 +1,9 @@
 # Introduction
 
-[![Build Status](https://travis-ci.org/Kashoo/synctos.svg?branch=master)](https://travis-ci.org/Kashoo/synctos) [![npm version](https://badge.fury.io/js/synctos.svg)](https://badge.fury.io/js/synctos) [![dependencies Status](https://david-dm.org/Kashoo/synctos/master/status.svg)](https://david-dm.org/Kashoo/synctos/master) [![devDependencies Status](https://david-dm.org/Kashoo/synctos/master/dev-status.svg)](https://david-dm.org/Kashoo/synctos/master?type=dev)
+[![Build Status](https://travis-ci.org/Kashoo/synctos.svg?branch=master)](https://travis-ci.org/Kashoo/synctos)
+[![npm version](https://badge.fury.io/js/synctos.svg)](https://www.npmjs.com/package/synctos)
```

Comment:
The notable change here is that the "npm version" badge's target URL now goes directly to the npm package page. Also added line breaks between badges to improve readability when viewing the raw source of the README; it does not affect formatting when viewing the formatted README through GitHub's UI.